packages/clerk-js/src/core/clerk.ts

```diff
@@ -1071,6 +1046,34 @@ export default class Clerk implements ClerkInterface {
     }
   };
 
+  #setIntermediateState = () => {
+    this.session = undefined;
+    this.organization = undefined;
+    this.user = undefined;
+    this.#emit();
+  };
+
+  #getLastActiveOrganizationFromSession = () => {
+    const orgMemberships = this.session?.user.organizationMemberships || [];
+    return (
+      orgMemberships.map(om => om.organization).find(org => org.id === this.session?.lastActiveOrganizationId) || null
+    );
+  };
+
+  #setSessionAndOrganization = (session?: ActiveSessionResource | null, organization?: OrganizationResource | null) => {
```

Comment:
❓ We never call this with an org as the 2nd argument, do we actually need the 2nd argument? Also, this seems to always be setting the user as well. Should that be reflected in the name somehow (either explicitly or by making the name more generic)?

Reply:
you are correct! i will remove organization. Any suggestion about the name?

Reply:
we discussed it offline and settled for `setAliases`